src/main/java/org/openhab/binding/zwave/handler/ZWaveSerialHandler.java

```diff
@@ -44,15 +51,22 @@
  *
  * @author Chris Jackson - Initial contribution
  */
+@NonNullByDefault
 public class ZWaveSerialHandler extends ZWaveControllerHandler {
 
+    private static final int SERIAL_RECEIVE_TIMEOUT = 250;
+    private static final long WATCHDOG_INIT_SECONDS = 5;
+    private static final long WATCHDOG_CHECK_SECONDS = 30;
+
     private final Logger logger = LoggerFactory.getLogger(ZWaveSerialHandler.class);
 
-    private SerialPortManager serialPortManager;
+    private final SerialPortManager serialPortManager;
 
-    private String portId;
+    private String portId = "";
```

Comment:
This ="" can be removed if we add this as required to the XML

Reply:
As the ZWaveSerialHandler class is now annotated with @NonNullByDefault, it is necessary to give the portId attribute an initial value, sorry.